micro-rdk/src/common/data_manager.rs

```diff
@@ -22,6 +22,8 @@ use futures_util::lock::Mutex as AsyncMutex;
 use prost::Message;
 use thiserror::Error;
 
+static MAX_SENSOR_CONTENTS_SIZE: usize = 64000;
```

Comment:
Why this number? Is it an esp32 constraint? something in app?

Reply:
Let's document it, and also consider cutting it to something smaller than the upper bound, like 32k or 16k.